### PR TITLE
REL-3368: Added 2018B config file for p1monitor.

### PR DESCRIPTION
**Pull request by sraaphorst**

### CODE REVIEW

**Comment by sraaphorst** [comment]
Regarding the `test_proposals`, it's always been this way, and confused me at first, too... in fact, I had to go look at @cquiroz's PRs for the 2018A semester to even remember how to do this, where he made the production file (there are no non-production semester files) `conf.production-2018A.xml` with `test_proposals` and then switched it to `proposals` when actually doing the release.

I think releasing the PIT is actually considerably more challenging than releasing the rest of OCS. The PIT code is cleaner, but there are some curious design choices and fiddly details with the p1-monitor and the backends.

Here's the 2018A file:
https://github.com/gemini-hlsw/ocs/blob/develop/bundle/edu.gemini.p1monitor/src/main/resources/edu/gemini/p1monitor/config/conf.production-2018A.xml

Here's the PR where you can see @cquiroz  changing it from `test_proposals` to `proposals` for release:
https://github.com/gemini-hlsw/ocs/commit/f197ae5e33285d2da346dac0c53d814049b94e5b#diff-ed39d62bdd3248ebf39d4036b25465b1

File: bundle/edu.gemini.p1monitor/src/main/resources/edu/gemini/p1monitor/config/conf.production-2018B.xml [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+        <to>pit.us@gemini.edu</to>
+        <template>us</template>
+    </type>
+
+    <type name="subaru">
+        <dir>/home/software/test_proposals/subaru/proposals/2018B</dir>
+        <to>pit.subaru@gemini.edu</to>
+    </type>
+
+    <filetranslation>
```
[the remark this answers is not in the document]
Comment (sraaphorst) [reply bundle/edu.gemini.p1monitor/src/main/resources/edu/gemini/p1monitor/config/conf.production-2018B.xml]: I have to admit that I don't have a clue what the file translations are for. I've never had to change them, so I've just let them be.

File: bundle/edu.gemini.p1monitor/src/main/resources/edu/gemini/p1monitor/config/conf.production-2018B.xml [hunk 2]
```diff
@@ -0,0 +1,169 @@
+<conf>
+    <smtp>smtp.hi.gemini.edu</smtp>
+
+    <bcc>swalker@gemini.edu</bcc>
+    <bcc>cquiroz@gemini.edu</bcc>
+    <bcc>sraaphorst@gemini.edu</bcc>
+
+    <type name="FastTurnaround">
+        <dir>/home/software/test_proposals/fast_turnaround/proposals/2018B</dir>
```
[the remark this answers is not in the document]
Comment (sraaphorst) [reply bundle/edu.gemini.p1monitor/src/main/resources/edu/gemini/p1monitor/config/conf.production-2018B.xml]: Sorry; I should have answered here. See my comment below.

**Comment by cquiroz** [comment]
Right, so we normally do a lot of test releases and switch to the production test for the final releases. In that sense this is fine. Some time I don't make a p1monitor PR until the las moment but it's ok to do it like this earlier in the process

**Comment by sraaphorst** [comment]
Thanks for clarifying that, @cquiroz... I wasn't sure if I needed to make a p1-monitor before the final release. I thought usually science wanted that to feel comfortable that proposals were being accepted properly.

**Comment by sraaphorst** [comment]
No worries. I don't know if we need to run with the test backends, but I suppose it might make science feel more confident.

**Pull request merged**